camel/kie-camel/src/test/java/org/kie/camel/component/BatchTest.java

```diff
@@ -608,14 +608,10 @@ public void testModifyObject() throws Exception {
                       stilton.getPrice() );
 
         //now test for code injection:
-        ModifyCommand.ALLOW_MODIFY_EXPRESSIONS = false;
```

Comment:
Does this mean that the value is now always present in the serialized/de-serialized content (xml)? Wouldn't that mean that caller can easily set it to true, enabling those potentially harmful modify expressions? It seems that the callee has no means to disable the modify expressions on their side (like was before the static field). Or is there some other mechanism to do that?